### ISSUE TITLE
Question: should copyright files be copied as is?

### ISSUE THREAD

**Issue by cjdcordeiro** [canonical/chisel#13]
When slicing a package, we'll most likely also want to make sure the corresponding copyright information is kept.

Our options are:
 1. simply install the DEB's copyright file as-is, or
 2. take only the relevant files paragraphs (corresponding to the installed files) of the DEB's copyright file. (**update:** it is not clear to me whether files paragraphs always refer to _source_ files, or whether they can also refer to subset of license-compliant files which are being shipped within the DEB)

Number 2. seems more useful for performing post-build source pedigree analysis, as we'll only get what we actually have. On the other hand, by modifying the copyright file, we're not staying true to the DEB...

Question originated by: https://github.com/ubuntu-rocks/dotnet/issues/28

**Comment by cjdcordeiro** [comment]
#### update

according to https://www.debian.org/doc/packaging-manuals/copyright-format/1.0, the copyright's `Files: ` directive 

> matches any file ... in the source tree...

> ... match pathnames that start at the root of the source tree
.

There's a catch-all pattern where

> a single paragraph with Files: * can be used to state the license and copyright for the whole package

which works as an alternative for the simplified licensing declaration which can be defined by the `License:` directive within the [header paragraph](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#header-paragraph).


---

in summary, it is safe to assume that files paragraphs always refer to the source tree, and thus it would be hard (and probably questionable) to try to infer which of the DEB's contents were generated by which parts of the source tree.

Let's just take the package's copyright file as a whole. That should fix https://github.com/ubuntu-rocks/dotnet/issues/28 for now
